src/algebra/category/Group.lean

```diff
@@ -84,3 +85,85 @@ instance has_forget_to_CommMon : has_forget₂ CommGroup CommMon :=
 induced_category.has_forget₂ (λ G : CommGroup, CommMon.of G)
 
 end CommGroup
+
+
+variables {X Y : Type u}
+
+section
+variables [group X] [group Y]
+
+/-- Build an isomorphism in the category `Group` from a `mul_equiv` between `group`s. -/
+@[to_additive add_equiv.to_AddGroup_iso "Build an isomorphism in the category `AddGroup` from a `add_equiv` between `add_group`s."]
```

Comment:
Does `simps` get along with `to_additive`?

Reply:
AFAIR, `to_additive` doesn't copy `simps` attribute. But it should be easy to add it.

Reply:
Sounds like a good idea (-;

Reply:
I found the interaction between `simps` and `to_additive` ... confusing! (Unfortunately there's a lot of recompiling between `to_additive.lean` and `Group.lean`.) In the end I've just gone with
```
attribute [simps] mul_equiv.to_Group_iso add_equiv.to_AddGroup_iso
```
which works great. If anyone would like to experiment further, please do. :-)